hushline/forms.py

```diff
@@ -25,5 +31,15 @@ def __call__(self, form: Form, field: Field) -> None:
             raise ValidationError(self.message)
 
 
+def is_valid_password_swap(
+    *, check_password: Callable[[str], bool], old_password: str, new_password: str
```

Comment:
I don't love passing a callable here, is the goal to keep all password validation logic in one part of the code?

Reply:
Yeah, that was the idea. New refactorings also made it unnecessary. I removed this pattern in the new PR #635